modules/openbsd/openbsd-driver.h

```diff
@@ -0,0 +1,13 @@
+#ifndef OPENBSD_SYSTEM_DRIVER_H_INCLUDED
+#define OPENBSD_SYSTEM_DRIVER_H_INCLUDED
+
+#include "driver.h"
+#include "logsource.h"
+
+struct OpenBSDDriver;
+
+LogDriver *openbsd_sd_new(gchar *filename,GlobalConfig *);
+
+LogSourceOptions *openbsd_sd_get_option(LogDriver *);
+
```

Comment:
I haven't send this one used. As this is a LogReader based source, it should be that anyway.

Reply:
Yes this is unused. I removed.